examples/use_config_ifp/CMakeLists.txt

```
@@ -5,8 +5,18 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/")
 set(ignition-find_config_DIR "${CMAKE_SOURCE_DIR}/")
 #ign_find_package(ignition-find_config) will search for Findignition-find_config.cmake before ignition-find_configConfig.cmake 
 ign_find_package(ignition-find_config)
+if(CONFIG_TEST MATCHES "The package was found using MODULE mode")
+    message(STATUS "The package was found using MODULE mode")
+else()  
+    message( SEND_ERROR "Couldn't find correct package in MODULE mode.\nign_find_package CONFIG test failed")
```

nit: remove whitespace next to opening parenthesis:
```suggestion
    message(SEND_ERROR "Couldn't find correct package in MODULE mode.\nign_find_package CONFIG test failed")
```